src/platform/forms-system/src/js/widgets/TextWidget.jsx

```diff
@@ -4,11 +4,21 @@ import PropTypes from 'prop-types';
 const numberTypes = new Set(['number', 'integer']);
 
 export default function TextWidget(props) {
+  const pageIndex = props.formContext?.pagePerItemIndex;
   let inputType = props.options.inputType;
   if (!inputType) {
     inputType = numberTypes.has(props.schema.type) ? 'number' : props.type;
   }
 
+  /**
+   * addIndex
+   * ariaDescribedby id may need to be indexed, add `:index` to the id and the
+   * formContext pagePerItemIndex will replace it
+   * @param {String|null} id - aria-describedby id with ":index" placeholder
+   */
+  const addIndex = (id = '') =>
+    id && typeof pageIndex !== 'undefined' ? `${id}${pageIndex}` : id;
```

Comment:
But here there isn't an underscore separating the id from the index - that's what I was trying to get at last time. I think having an underscore in here would make sense, but other than that I have no complaints.

Reply:
Done. Thanks!